src/udev/udev-builtin-input_id.c

```diff
@@ -211,6 +211,8 @@ static bool test_pointers(struct udev_device *dev,
                 has_joystick_axes_or_buttons = test_bit(button, bitmask_key);
         for (button = BTN_TRIGGER_HAPPY1; button <= BTN_TRIGGER_HAPPY40 && !has_joystick_axes_or_buttons; button++)
                 has_joystick_axes_or_buttons = test_bit(button, bitmask_key);
+        for (button = BTN_DPAD_UP; button <= BTN_DPAD_RIGHT && !has_joystick_axes_or_buttons; button++)
+                has_joystick_axes_or_buttons = test_bit(button, bitmask_key);
```

Comment:
The CentOS CI seems to have been broken by this commit. In https://ci.centos.org/job/systemd-pr-build/3403/console I see the following error
```
src/udev/udev-builtin-input_id.c: In function 'test_pointers':
src/udev/udev-builtin-input_id.c:214:46: error: 'BTN_DPAD_RIGHT' undeclared (first use in this function)
         for (button = BTN_DPAD_UP; button <= BTN_DPAD_RIGHT && !has_joystick_axes_or_buttons; button++)
                                              ^
src/udev/udev-builtin-input_id.c:214:46: note: each undeclared identifier is reported only once for each function it appears in
  CC       src/resolve/test-dnssec-complex.o
make[2]: *** [src/udev/libudev_core_la-udev-builtin-input_id.lo] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [all-recursive] Error 1
make: *** [all] Error 2
```